src/main/java/com/iexec/sms/tee/config/SideComputeInternalConfiguration.java

```diff
@@ -0,0 +1,58 @@
+package com.iexec.sms.tee.config;
+
+import com.iexec.sms.api.config.TeeAppConfiguration;
+import org.springframework.beans.factory.annotation.Value;
+import org.springframework.context.annotation.Bean;
+import org.springframework.context.annotation.Configuration;
+import org.springframework.util.unit.DataSize;
+
+import javax.validation.constraints.NotBlank;
+import javax.validation.constraints.Positive;
+
+@Configuration
+public class SideComputeInternalConfiguration {
+    @Bean
+    TeeAppConfiguration preComputeConfiguration(
+            @Value("${tee.workflow.pre-compute.image}")
```

Comment:
```suggestion
            @Value("${tee.worker.pre-compute.image}")
```

Reply:
Please see https://github.com/iExecBlockchainComputing/iexec-sms/pull/119/commits/a305d6436fcd58eae4590abba97f1d9e560eab5b.

Reply:
Thanks :)

Reply:
Maybe you can remove all `workflow` occurrences of this PR?

Reply:
Sure, please see https://github.com/iExecBlockchainComputing/iexec-sms/pull/119/commits/488bc010e370f2bcdce39bc5f38371751afa47b9. It removes the `workflow` values from the integration tests property file and cleans the integration tests, so that it is easier to mock only some parts of the SMS.